examples/platform/silabs/efr32/rs911x/rsi_wlan_config.h

```diff
@@ -53,20 +53,29 @@
 #endif
 
 //! To set Extended custom feature select bit map
-#if WIFI_ENABLE_SECURITY_WPA3
+#if WIFI_ENABLE_SECURITY_MODE_WPA3_TRANSITION /* TRANSITION MODE:WPA2 AND WPA3 SUPPORTED */
+#ifdef CHIP_9117
+#error "WPA3 tranisiotion mode was unsupported for 9117"
+#endif /* CHIP_9117 */
+#ifdef RSI_M4_INTERFACE
```

Comment:
`RSI_M4_INTERFACE` can be removed
The same configuration will work for both **WPA3 ONLY** and **WPA3 TRANSITION** which is to say
```c
...
#if (WIFI_ENABLE_SECURITY_MODE_WPA3_TRANSITION || WIFI_ENABLE_SECURITY_MODE_WPA3_ONLY)
#define RSI_EXT_CUSTOM_FEATURE_BIT_MAP (EXT_FEAT_320K_MODE | EXT_FEAT_IEEE_80211W)
#else
...
```
should work